### ISSUE TITLE
Remove the VOLUME directive from Dockerfile

### ISSUE THREAD

**Issue by ianroberts** [jlesage/docker-firefox#107]
Currently the `Dockerfile` includes the directive:

```
VOLUME ["/config"]
```

This is un-necessary since anyone following the documentation will manually specify a `-v` option to `docker run`, and the directive means that if you run the container _without_ an appropriate `-v` then it will silently create a persistent volume in the background which the user must know to manually delete.  It is not possible to start a completely ephemeral container with no persistent storage.

Removing the `VOLUME` directive would not have any negative effects on current users who are following the documentation, but it would open up use cases like a throwaway web browser to explore potentially dangerous links, where you don't _want_ to persist the config directory.

**Comment by jlesage** [comment]
The `VOLUME` directive is also very useful for another reason: it is used by NAS OSes like unRAID, Synolog, QNAP or by container management softwares like Portainer to propose users semi-automatic configuration of the container.

> then it will silently create a persistent volume in the background which the user must know to manually delete

I don't see this behaviour.  If I run `docker volume ls` before and after temporarily creating and running a container (`docker run --rm jlesage/firefox`), I don't see additional volumes.

**Comment by ianroberts** [comment]
Doing `docker run --rm ...` is roughly equivalent to `docker run ...` followed by `docker rm -v <container>`, which does indeed remove anonymous volumes, but a `docker run` (without the `--rm`) followed later by a `docker rm` will leave the volume dangling, as will `docker-compose` up followed by down if the compose file does not explicitly mount the volume.

**Comment by jlesage** [comment]
Yes you are correct.   And the docker-compose down operation also supports the `-v` option.

So by using the correct commands, someone can create a throwaway browser without creating orphan volumes.
